apache/superset
# feat: Adds the ECharts Sunburst chart

### SUMMARY
Completes the work made in https://github.com/apache/superset/pull/21625 to add the ECharts Sunburst chart. 

- Adds a thumbnail and example pictures
- Adds a Storybook entry for the chart
- Enables drill to detail by hierarchies
- Makes the relationship between hierarchies and the circles clear to the user
- Uses the [callback function](https://echarts.apache.org/en/option.html#tooltip.formatter) and returns an HTML element for the tooltip using the theme object
- Improves the tooltip description to show the hierarchy names
- Improves tooltip positioning as it is sometimes being cut off the screen
- Registers the chart again so it's available for use

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<img width="1166" alt="Screenshot 2023-01-23 at 5 10 49 PM" src="https://user-images.githubusercontent.com/70410625/214163134-87d4e6bb-62f0-489a-9ece-71aefb5ce28a.png">
<img width="1079" alt="Screenshot 2023-01-24 at 11 54 26 AM" src="https://user-images.githubusercontent.com/70410625/214357049-a402c7b5-f65a-4dd8-a32e-d5c4fd3a5329.png">
<img width="958" alt="Screenshot 2023-01-25 at 11 51 44 AM" src="https://user-images.githubusercontent.com/70410625/214626885-cad4b4ab-79a2-4d2a-85bc-ea0409a09ddd.png">

### TESTING INSTRUCTIONS
- Check the plugin description in the chart viz control
- Create a Sunburst v2 chart 
- Interact with the available controls in Explore
- Make sure all controls are working
- Check the plugin Storybook

### ADDITIONAL INFORMATION
